docs/docs/active-learning/acquisition-functions.md

```diff
@@ -39,6 +39,25 @@ On the following scenarios, uncertainty-based acquisition functions must be used
 - For convolutional neural networks, small, seemingly meaningless perturbations in the input space can completely change predictions.
 :::
 
+## Diversity-based acquisition functions {#diversity-based-acquisition-function}
+
+Diversity sampling is an active learning strategy that aims to ensure that the labeled training set includes a broad range of examples from across the input space. The underlying assumption is that a diverse set of training examples will allow the model to learn a more generalized representation, improving its performance on unseen data.
+
+In contrast to uncertainty-based methods, which prioritize examples that the model finds difficult to classify, diversity-based methods prioritize examples based on their novelty or dissimilarity to examples that are already in the training set. This can be particularly useful when the input space is large and the distribution of examples is uneven.
+
+We include the following diversity-based acquisition function:
+
+- [Image Diversity][ea-acquisition-function-image-diversity]: This metric clusters the images according to number of classes in the 
+ontology file. Then, it chooses samples from each cluster one-by-one to form an equal number of samples from 
+each cluster. Samples are chosen according to their proximity to cluster centroids (closer samples will be 
+chosen first).
+
+:::tip
+Diversity-based acquisition functions are generally easier to use compared to the uncertainty-based functions because 
+they may not require any ML model. See [diversity-sampling-on-unlabeled-data example](../tutorials/diversity-sampling-on-unlabeled-data.mdx) to 
```

Comment:
can you put the references in the bottom of the file so we only update them there when they change? like the examples at the bottom of the file